soh/src/code/z_vr_box.c

```diff
@@ -446,23 +446,17 @@ void func_800AF178(SkyboxContext* skyboxCtx, s32 arg1) {
 
 void LoadSkyboxTex(GlobalContext* globalCtx, SkyboxContext* skyboxCtx, int segmentIndex, int imageIndex, char* tex, int width, int height, int offsetW, int offsetH)
```

Comment:
Can you remove the `globalCtx` argument since it no longer goes used.

Reply:
Yes, sure. I was thinking about keeping the other parameters (width, height etc.) since they could be nice for debugging and maybe we need them later for some reason. Is that ok?

Reply:
Sure.